cmd/crdvalidator/manifests/04_webhook.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: crd-validation-webhook # TODO (tylerslaton): do we want this domain?metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: default/crd-validation-webhook-certificate
+webhooks:
+  - name: "crd-validation-webhook.acme.com"
```

Comment:
WDYT about "crd-validation-webhook.rukpak.io"? I know we want to keep this webhook as it's own component, but we are housing it in rukpak initially and that's its origin essentially.

Reply:
Maybe it makes sense to create its own namespace? Potentially something like `crd-validation-webhook.io`. My main hesitation with doing anything `rukpak` specific to it is that we'll have to undo it later. Granted, its trivial work to fix so I'm fine either way.

Reply:
Added the new address in the most recent commit.